ci/publish-crate.sh

```diff
@@ -19,9 +19,12 @@ CRATES=(
   metrics
   client
   drone
-  programs/{budget_api,config_api,storage_api,token_api,vote_api}
+  instruction-processors/vote_api
```

Comment:
ugh so sorry for the churn.  instruction-processors are now progams again.

https://media.tenor.com/images/a4ace8088e197c934c8506244a2106cb/tenor.gif